app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+# Handle micropost CRUD.
+class MicropostsController < ApplicationController
+  before_action :set_micropost, only: %i[show edit update destroy]
+
+  # GET /microposts
+  # GET /microposts.json
+  def index
```

Comment:
Our controller actions get 2 comments. The first shows what the request looks like, the second describes what it does. For this one, it would look like
```
# GET /microposts.
# List microposts.
```

Reply:
Apply this same pattern to all controller actions. Look through IO and DM controllers for more examples if you want.